### ISSUE TITLE
How to install linux mint cinnamon 18 Sarah on Chromebook 2016 using crouton?

### ISSUE THREAD

**Issue by tropivan** [dnschneid/crouton#2933]
Hi David!

I just opened the box of a brand new Asus Chromebook C202SA-YS02, board name TERRA, base board strago, kernel 3.18, platform Braswell, form factor Chromebook.

I got into Dev-Mode, installed crouton and crouton extension and finally I followed your steps for:
"The easy way (assuming you want an Ubuntu LTS with Xfce)"
(https://github.com/dnschneid/crouton/blob/master/README.md)

Everything is working fine, even installing VLC, libreoffice and firefox on that Ubuntu 4.8 Xfce linux... but this computer is for a 70 years old friend who is currently running Linux Mint 18 Sarah on his previous computer...

So, I would really like to wipe out Chromium from the small 16gb sd internal card and install Linux Mint 18 Sarah on it for him!

Would you please share any guidelines to achieve that?

Best regards fro Costa Rica,

tropivan

**Comment by DennisLfromGA** [comment]
@tropivan,

I am definitely *not* attempting to speak for David but I thought I'd offer another suggestion instead of blowing away CrOS if I may. 

There is a project called ['chrx'](https://github.com/reynhout/chrx) that is gaining in popularity and might suit your needs. It will let you keep CrOS and dual-boot into one of a number of different distros. They will work on your Braswell Chromebook with a little work - all done in the 'chrx' scripts.

CrOS only uses about 7GB so on a 16GB SSD you'll have about 9GB left for other things.
Checkout the ['chrx sample measurements](https://github.com/reynhout/chrx#user-content-sample-measurements) for the distros and sizes available.

It does require that the Chromeboook be in Developer mode, and stay there :) so it could be a sticking point with a senior. I'm definitely a senior too, just shy of 70 and I can do it. :)

*Hope this helps,*
-**DennisL**

**Comment by tropivan** [comment]
Thank you @DennisLfromGA for your time responding me and for your recomendations!

I will check both links.

My friend would prefer to not have Chromium at all in the computer and to just have Linux Mint on it... I have read about pros & cons when Chromium is away, mostly with drivers for WiFi, touchpad and keyboard, also firmwares... I will back with news about the experiments here...

Regards from Costa Rica,

tropivan

**Comment by DennisLfromGA** [comment]
@tropivan,

Sounds good, can't wait to hear about your experiences.

I have a Dell Chromebook 13 which has the advantage of an upgradable SSD so mine has a 256GB SSD and dual-boots Linux Mint 18 (Sarah). It's still in Developer mode but when I want to boot into CrOS I press <kbd>ctrl</kbd> + <kbd>D</kbd> and when I want to boot into Mint I press <kbd>ctrl</kbd> + <kbd>L</kbd>, pretty nifty.

*Hope this helps,*
-**DennisL**

**Comment by tropivan** [comment]
Good night @DennisLfromGA 

I tried today a different target and release with crouton downloader, I don't have internet at home (Just a quarter of 1mb at my smartphone) so I'm using another friend's internet conection... The speed in Costa Rica... In the mountains... Is very slow, so it is still running the installation (more than 10hours?) hopefully I will know tomorrow if it worked!

The example from David was to download xfce (that installed Xubuntu), I need to achieve installing Linux Mint Sara, so I modified the crouton sudo and added -r xenial -t xfce

I will tell you the full sudo tomorrow when I back to my friend's Office to have my paper notes on sight... I reserve my brain memory for life instead than numbers and codes...

Best regards,

tropivan

**Comment by tropivan** [comment]
Greetings @DennisLfromGA

The last code I tried on the Asus Chromebook C202SA-YS02 in order to install Linux Mint was:

sudo sh -e ~/Downloads/crouton -r xenial -t xfce-Desktop

Crouton accept the code and the installation was running for 2 days... finally the installation completed successfully but I no idea how to execute it... I tried:

sudo startxxenial ,but nothing happen... then I tried:

sudo enter-chroot startxxenial and sudo enter-chroot startxfce ,nothing happen again...

I continue with the proceed to remove the write protection screw from the mother board to erase the flashcard, but the Braswell on this chromebook is not listed as compatible with SeaBoot... 

Finally I quit!

3 weeks trying without success was enough time to buy a Lenovo Ideapad 110S with Win10, wipe out Win10 and installed Linux Mint 18 Sarah easy as always!!!

My friend as happy with the new toy.

Asus Chromebook C202SA-YS02 have 4gb RAM and 16gb eMMC flashcard... everything very welded on board... no upgrade possibles...

Lenovo Ideapad 110S have 2gb RAM and 32gb eMMC flashcard, but the RAM is mounted on slots so it could be upgraded to 4gb and there is a slot for sata ngff solid state drive up to 256gb!!!

Lenovo wons!

Asus has SD card slot for memory expansion, Lenovo has microSD card for same idea.

That is the end of this story... by now... hopefully some day a visioneer brain like David Schneider o John Lewis will bring to us a solution to install Linux Mint 18 or later on any Intel/AMD chip based Chromebook!

Enjoy your holidays and thank you for your support and time!

Regards from Costa Rica,

tropivan
